### ISSUE TITLE
Only color is provided to differentiate between link and normal text_A11y_Fabric Notes_CosmicNotes_Usable.

### ISSUE THREAD

**Issue by shindepra** [microsoft/fabricnotes#35]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHub Tags:
#A11yTCS; #A11yUsable; #DesktopWeb; #Win11; #A11ySev4; #Fabric Notes #CELA P3; #Feature_FabricNotes_Web_APR2024. 

### Environment and OS details:
Windows Edition: - Windows 11 Enterprise (23H2)
OS build: - 22631.3007
Chromium Edge Version: Version 123.0.2420.65 (Official build) dev (64-bit) 

### Repro Steps:

1. Hit the URL: [[Fabric Notes - Simple drawings illustrating the main concepts of Microsoft Fabric](https://microsoft.github.io/fabricnotes/) Fabric Notes page will open. 
3. Tab till the "Fabric Notes" Text and press enter. 
4. Tab till 'Cosmic Notes'.
5. Observe that no Underline is provided for 'Cosmic Notes' Link.

### Actual Result:
Only color is provided to differentiate between link and normal text.

### Expected Result:
The under should be provided to differentiate link form the normal text. 

### User Impact:
Only color is not sufficient to differentiate between link and normal text. Color blind user will not able to differentiate the link and normal text. 

### Attachment:
<img width="949" alt="image" src="https://github.com/microsoft/fabricnotes/assets/143686145/219a1d75-6345-4e63-9745-2a28855ebda7">

**Comment by Yash14j** [comment]
GitHub Tags:#Rev:yaja;

**Comment by shindepra** [comment]
Verified the bug in below environment,
URL: [[Fabric Notes - Simple drawings illustrating the main concepts of Microsoft Fabric](https://microsoft.github.io/fabricnotes/)
Windows Edition: - Windows 11 Enterprise (23H2)
OS build: - 22631.3007
Chromium Edge Version: Version 123.0.2420.65 (Official build) dev (64-bit)
Issue is fixed.
<img width="800" alt="image" src="https://github.com/microsoft/fabricnotes/assets/143686145/2b72e1ab-e8c2-4431-9da9-b4859016d824">
